<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 89, 79);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 53, 47);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>FORM PEMINJAMAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblPinjam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>581</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID Peminjaman</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID User</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID Laptop</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tanggal Peminjaman</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tanggal Pengembalian</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 89, 79);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Form Peminjaman</string>
    </property>
    <widget class="QPushButton" name="btnSimpan">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 220, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 220, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Laptop ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PeminJaman ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUID">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editLID">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPID">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>PID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TGL Peminjaman</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TGL Pengembalian</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="datePinjam">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateBalik">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUID">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 220, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Cari</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLID">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 220, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Cari</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBaru">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 220, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Baru</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
